app/gui/src/project-view/components/ComponentDocumentation.vue

```diff
@@ -19,20 +19,19 @@ function docsForSelection() {
   return Ok(suggestionId)
 }
 
-const displayedId = computed(() =>
-  props.displayedSuggestionId != null ? Ok(props.displayedSuggestionId) : docsForSelection(),
-)
+const docs = computed(() => docsForSelection())
+// When the selection changes, we cancel the displayed suggestion override that can be in place.
+watch(docs, (_) => (displayedId.value = null))
```

Comment:
Setting `displayId.value` to `null` to make it reflect `docs.value` (which isn't necessarily `null` here) is confusing. Why not reset `overrideDisplayed` directly?

Reply:
That’s an oversight, thank you